Mathlib/MeasureTheory/Measure/MeasureSpace.lean

```diff
@@ -1631,6 +1631,15 @@ protected lemma add (h1 : μ₁ ≪ ν) (h2 : μ₂ ≪ ν') : μ₁ + μ₂ ≪
   simp only [add_toOuterMeasure, OuterMeasure.coe_add, Pi.add_apply, add_eq_zero] at hs ⊢
   exact ⟨h1 hs.1, h2 hs.2⟩
 
+lemma add_left_iff (μ₁ μ₂ ν : Measure α) :
```

Comment:
```suggestion
lemma add_left_iff {μ₁ μ₂ ν : Measure α}
```
When variables can be deduced from both sides of an iff, they are typically implicit, as they can be obtained automatically when one rewrites in one direction or the other.